mne/source_estimate.py

```diff
@@ -359,10 +360,33 @@ def read_source_estimate(fname, subject=None):
     return stc
 
 
-def _make_stc(data, vertices, tmin=None, tstep=None, subject=None,
+def _get_src_type(src, vertices):
+    src_type = None
+    if src is None:
+        warn_("src should not be None for have a robust guess of stc type.")
+        if isinstance(vertices, list) and len(vertices) == 2:
+            src_type = 'surf'
+        elif isinstance(vertices, np.ndarray) or isinstance(vertices, list)\
+                and len(vertices) == 1:
+            src_type = 'vol'
+        elif isinstance(vertices, list) and len(vertices) > 2:
+            src_type = 'mixed'
+    else:
+        src_types = set([ss['type'] for ss in src])
```

Comment:
this should just be `src.kind`

https://github.com/mne-tools/mne-python/blob/master/mne/source_space.py#L207

Reply:
so you suggest to change src.kind to 'mixed' and not 'combined' ? that means a deprecation cycle ...

Reply:
I think it's a bug it was ever `combined`